.lune/util/generator/types.luau

```diff
@@ -0,0 +1,39 @@
+local Types = {}
+
+local mappedLuauDataTypes = {
+	-- generic Roblox datatypes
+	[{ "nil" }] = "https://www.lua.org/pil/2.1.html",
+	[{ "boolean", "bool" }] = "https://www.lua.org/pil/2.2.html",
+	[{ "number" }] = "https://www.lua.org/pil/2.3.html",
+	[{ "string" }] = "https://www.lua.org/pil/2.4.html",
+	[{ "table" }] = "https://www.lua.org/pil/2.5.html",
+	[{ "tuple", "..." }] = "https://www.lua.org/pil/5.1.html",
+	[{ "userdata", "proxy" }] = "https://www.lua.org/pil/28.1.html",
+}
```

Comment:
Can we expand this with Luau's types like buffer and vector? The "Roblox" comment seems unnecessary too.

Reply:
ok, yeah i didn't bother updating, just brought over from discord-luau